infrastructure/network-explorer/network-explorer-setup.rst

```diff
@@ -274,6 +274,18 @@ The following example uses 4 shards per reducer stage.
           matchingShards: 4
           aggregationShards: 4
 
+Estimating Reducer CPU and Memory Usage
++++++++++++++++++++++++++++++++++++++++
+To estimate the CPU and Memory usage the reducer may require from a node, you can use these simple formulas:
+
+::
+
+    Memory in Mebibytes (Mi) = 4 * Number of nodes in your cluster + 60
+    Fractional CPU in milliCPU (m) = Number of nodes in your cluster + 30
+
+This gives you an appromixate expected usage.  You may want to multiple the final numbers by a factor of 1.5 or 2 to give headroom for growth and spikes in usage.
```

Comment:
Extra space after the first sentence. Also change the second sentence to "Multiply the final numbers..."